### PR TITLE
hw-mgmt: thermal: Add new systems support to TC

### PR DESCRIPTION
**Pull request by sholeksandr**

### CODE REVIEW

File: usr/etc/hw-management-thermal/tc_config_qm3400.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		"sensor_amb":     {"pwm_min": 30, "pwm_max" : 50, "val_min": 30000, "val_max": 55000, "poll_time": 30},
+		"voltmon\\d+_temp":{"pwm_min": 30, "pwm_max" : 100, "val_min": "!85000", "val_max": "!125000", "poll_time": 60},
+		"sodimm\\d_temp" :{"pwm_min": 30, "pwm_max" : 70, "val_min": "!70000", "val_max": 95000, "poll_time":  60},
+		"drivetemp" :{"pwm_min": 30, "pwm_max" : 70, "val_min": "!70000", "val_max": 95000, "poll_time": 60},
+		"connectx8" :{"pwm_min": 30, "pwm_max" : 100, "val_min": "70000", "val_max": 105000, "poll_time": 3}
+	},
+	"sensor_list" : ["asic1", "connectx8", "cpu", 
+	"drwr1", "drwr2", "drwr3", "drwr4", "drwr5",
+	"psu1", "psu2", "sensor_amb", "ibc1", "ibc2", "ibc3", "ibc4", "sodimm1",
+	"voltmon1", "voltmon2", "voltmon3", "voltmon4", "voltmon5", "voltmon6", "voltmon7", "voltmon8", "voltmon9", "voltmon10", "voltmon11"]
```
Comment (ciju-nvidia) [inline usr/etc/hw-management-thermal/tc_config_qm3400.json:37]: There will be 4 PUSs in crocodile.

File: usr/etc/hw-management-thermal/tc_config_qm3400.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	},
+	"psu_fan_pwm_decode" : {"0:100": -1},
+	"fan_trend" : {
+        "C2P": {
+			"0" : {"rpm_min":6879, "rpm_max":21800, "slope": 213.2, "pwm_min" : 30, "pwm_max_reduction" : 10, "rpm_tolerance" : 30},
+			"1" : {"rpm_min":5650, "rpm_max":20500, "slope": 200.4, "pwm_min" : 30, "pwm_max_reduction" : 10, "rpm_tolerance" : 30}
+			}
+	}, 
+	"dev_parameters" : {
+		"asic":           {"pwm_min": 30, "pwm_max" : 100, "val_min":"!70000", "val_max":"!105000", "poll_time": 3},
```
Comment (ciju-nvidia) [inline usr/etc/hw-management-thermal/tc_config_qm3400.json:25]: There are more than 1 asic in Crocodile. May be add "asic\\d+" ?

File: usr/etc/hw-management-thermal/tc_config_qm3400.json (hunk 1 above)
Comment (ciju-nvidia) [inline usr/etc/hw-management-thermal/tc_config_qm3400.json:37]: There are only 6 voltage monitors.

Comment (sholeksandr) [reply usr/etc/hw-management-thermal/tc_config_qm3400.json:37]: No, at hw-spec Figure 33 i see 11 in voltmon table

**Pull request closed without merging**